dozer-ingestion/aerospike/src/connector.rs

```diff
@@ -389,6 +429,8 @@ impl Connector for AerospikeConnector {
                                         name: name.clone(),
                                         typ: if name == "inserted_at" {
                                             FieldType::Timestamp
+                                        } else if name == "PK" {
```

Comment:
PK are not necessarily UInt

Reply:
Yeah, it can be other types. To handle that user should use schema.json.

Reply:
By default, aerospike doesn't actually store the keys, so hitting this path will be rare anyway

Reply:
Yeah. It is more for testing purposes